Update docs for RN SDK v5 (#6230)
getsentry/sentry-docs: src/platform-includes/getting-started-install/react-native.mdx

yarn add @sentry/react-native
```

Run `@sentry/wizard`:

```bash
npx @sentry/wizard -i reactNative -p ios android

cd ios
pod install
```

<Note>

For React Native 0.68 and older you will have to manually patch the Xcode project. Read more at the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.

</Note>

[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [setup manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You need to do this only once, then the created files can go into your version control system.

> 

Author: Thanks for all the suggestions. 

With the `can` here, I wanted to express that people can add there to their VSC but they don't have to.

Does `will` mean the same thing here?

> 

Reviewer: To give people more of a feeling of choice, I'd suggest being more explicit about it and saying something like: "If you want to add files to your VCS, you need to only add them once, they will then go into your version control system."

Otherwise, saying "can" instead of "will" makes it sounds like we're not sure where the files will go.

> 

Reviewer 2: I believe what Krystof is trying to say, is something along these lines: "You only need to patch the files once. Then you can add them to your version control system."

> [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [setup manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You need to do this only once, then the created files can go into your version control system.
> 

Author: ```suggestion
[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [set up manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You only need to patch the project once. Then you can add the patched files to your version control system.
```

@lizokm Would this work?

The following changes will be performed by Sentry Wizard:

- enable the Sentry React Native Gradle build step for Android to auto upload generated source maps and debug symbols
- wrap _Bundle React Native code and images_ Xcode project build phase script to upload generated source maps and collect bundled node modules
- add _Upload Debug Symbols to Sentry_ Xcode project build phase
- store build credentials in _*ios/sentry.properties*_ and _*android/sentry.properties*_.
- configure Sentry for the supplied DSN in your _*App.tsx*_ file

> - wrap your _App_ for [touch event tracking](/platforms/react-native/touchevents/) and [automatic performance monitoring](/platforms/react-native/performance/instrumentation/automatic-instrumentation/)

Reviewer 3: Is this done by Wizard as well? TIL

Author: Actually my bad. It's not. :( But would be cool.

> - enable the Sentry React Native Gradle build step for Android to auto upload generated source maps and debug symbols
> - wrap _Bundle React Native code and images_ Xcode project build phase script to upload generated source maps and collect bundled node modules
> - add _Upload Debug Symbols to Sentry_ Xcode project build phase
> - store build credentials in _*ios/sentry.properties*_ and _*android/sentry.properties*_.
> - configure Sentry for the supplied DSN in your _*App.tsx*_ file
> - wrap your _App_ for [touch event tracking](/platforms/react-native/touchevents/) and [automatic performance monitoring](/platforms/react-native/performance/instrumentation/automatic-instrumentation/)

Reviewer: ```suggestion
- Enable the Sentry React Native Gradle build step for Android to auto-upload generated source maps and debug symbols.
- Wrap the _Bundle React Native code and images_ Xcode project build phase script to upload generated source maps and collect bundled node modules.
- Add _Upload Debug Symbols to Sentry_ Xcode project build phase.
- Store build credentials in _*ios/sentry.properties*_ and _*android/sentry.properties*_.
- Configure Sentry for the supplied DSN in your _*App.tsx*_ file.
- Wrap your _App_ for [touch event tracking](/platforms/react-native/touchevents/) and [automatic performance monitoring](/platforms/react-native/performance/instrumentation/automatic-instrumentation/).
```

### Expo

> If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.

Reviewer: ```suggestion
If you're using Expo, read about [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.
```
